diff '--color=auto' -Naur ncurses-6.4.orig/configure ncurses-6.4/configure
--- ncurses-6.4.orig/configure	2025-04-09 14:15:29.995158389 -0400
+++ ncurses-6.4/configure	2025-04-09 14:17:12.728151649 -0400
@@ -3481,7 +3481,7 @@
 
 		save_CPPFLAGS="$CPPFLAGS"
 		eval cf_includedir=${includedir}
-		CPPFLAGS="$CPPFLAGS -I${cf_includedir}"
+		CPPFLAGS="$CPPFLAGS"
 
 cat >"conftest.$ac_ext" <<_ACEOF
 #line 3487 "configure"
@@ -6386,7 +6386,7 @@
 		fi
 		cf_cv_rm_so_locs=yes
 		;;
-	(linux*|gnu*|k*bsd*-gnu)
+	(linux*|gnu*|k*bsd*-gnu|nto-qnx*)
 		if test "$DFT_LWR_MODEL" = "shared" && test -n "$LD_RPATH_OPT" ; then
 			LOCAL_LDFLAGS="${LD_RPATH_OPT}\$(LOCAL_LIBDIR)"
 			LOCAL_LDFLAGS2="$LOCAL_LDFLAGS"
@@ -6564,7 +6564,7 @@
 		CC_SHARED_OPTS=
 		MK_SHARED_LIB='${LD} -Wshared -Wxld=-export_all -o $@'
 		;;
-	(nto-qnx*|openbsd*|freebsd[12].*)
+	(openbsd*|freebsd[12].*)
 		CC_SHARED_OPTS="$CC_SHARED_OPTS -DPIC"
 		MK_SHARED_LIB='${LD} ${LDFLAGS} -Bshareable -o $@'
 		test "$cf_cv_shlib_version" = auto && cf_cv_shlib_version=rel
